### ISSUE TITLE
Add URL of spec/draft to each feature

### ISSUE THREAD

**Issue by meetwudi** [compat-table/compat-table#468]
It is hard to follow-up a new feature in ES6 but now I found compat-table can do this for me. What about add URL of spec/draft to each feature?

**Comment by webbedspace** [comment]
Hm, the thing is, more than a handful of sub-tests actually do not map cleanly to a section of the spec. Like: "computed accessors" is really just the intersection of two distinct mutually permitted features.

**Comment by meetwudi** [comment]
I see. I just had hard time to confirm that :) I will close this issue.
